tracetools/CMakeLists.txt

```
@@ -80,6 +80,9 @@ if(TRACETOOLS_LTTNG_ENABLED)
   if(NOT TRACETOOLS_NO_RDYNAMIC)
     target_link_libraries(${PROJECT_NAME} "-rdynamic")
   endif()
+if(NOT WIN32)
```

Gah, this is my fault. I didn't test this change before I suggested it.

```suggestion
endif()
if(NOT WIN32)
```